### ISSUE TITLE
[Accessibility] Eco news:  ‘Filter by’ list has no name and doesn't keep the focus.

### ISSUE THREAD

**Issue by yuriiLisunov** [ita-social-projects/GreenCity#2330]
Actual:
‘Filter by’ list doesn't have a name and isn't focusable when using tab navigation.
The same issue for 'Filter by' on 'Add new habit' page.

Expected:
‘Filter by’ list should be named, be focusable and the name of the list should be read by SR when using tab navigation.
